includes/definitions/gaia.yaml

```diff
@@ -3,10 +3,17 @@ text: 'Check Point GAiA'
 type: firewall
 icon: checkpoint
 ifname: 1
+mib_dir:
+    - checkpoint
 over:
     - { graph: device_bits, text: 'Device Traffic' }
     - { graph: device_processor, text: 'Processor Usage' }
     - { graph: device_mempool, text: 'Memory Usage' }
 discovery:
     - sysObjectId:
         - .1.3.6.1.4.1.2620.1.6.123.1
+    - snmpget:
```

Comment:
Just FYI, this snmp get will be queried for every OS after this one (alphabetically)

It would be great if you could run a check before any snmpget to narrow it down a bit.

Reply:
Why will it be run for every os after this? Surely if it makes it this far it will just do it once then move on to the next OS?

I don't see any other way of identifying this OS though, it's using standard Linux sysDescr and sysObjectID :(

Reply:
No, it only does it once, but I'm saying if the target system is anything past G alphabetically, this snmp_get will be run.

In the snmprec file, you have sysObjectId and sysDescr, why not use those to only have the get run on linux systems?

Reply:
Can you even do that, I thought if you said check those two + this snmpget it will run the snmpget or will it return false once the first check against say sysObjectID fails if it's not Linux?